core/src/main/java/org/testcontainers/DockerClientFactory.java

```diff
@@ -178,22 +174,6 @@ private void checkMountableFile(DockerClient dockerClient, String id) {
         }
     }
 
-    private void checkExposedPort(String hostIpAddress, DockerClient dockerClient, String id) {
```

Comment:
this check is not needed anymore, Ryuk demands a connectivity hence implicitly checks for the exposed port